app/src/main/java/de/stephanlindauer/criticalmaps/Main.java

```diff
@@ -35,9 +35,12 @@
 
 public class Main extends AppCompatActivity implements NavigationView.OnNavigationItemSelectedListener {
 
+    private final static String KEY_NAVID = "main_navid";
```

Comment:
thanks for the comments. will change this one later globally in separate commit.